src/components/TodoItem.vue

```diff
@@ -1,23 +1,31 @@
 <template>
   <div class="todo">
     <div class="todo__header">
-      <router-link :to="`/details/${todo.id}/${JSON.stringify(todo)}`">
-        <h1
-          v-if="!isEditing"
-          class="todo__header_title"
-          :class="{ done: isDone }"
-        >
-          {{ todo.title }}
+      <div v-if="!isEditing">
+        <router-link v-if="!inDetailedMode" :to="`/details/${todo.id}`">
+          <h1 class="todo__header_title" :class="{ done: todo.done }">
```

Comment:
```suggestion
          <h1 class="todo__header_title" :class="{ done: isDone }">
```
Use computed property